src/mqttcm_webcfg.c

```diff
@@ -158,6 +165,7 @@ char * createMqttPubHeader(char * payload, ssize_t * payload_len)
 	char *pub_headerlist = NULL;
 
 	pub_headerlist = (char *) malloc(sizeof(char) * 1024);
+	memset(pub_headerlist, 0, sizeof(char) * 1024);
```

Comment:
Add Null check for pub_headerlist and then do memset

Reply:
added in new review